### DISCUSSION TITLE
HTTP Sink?

### DISCUSSION THREAD

**Discussion by canob** [opensearch-project/data-prepper#4073] [Q&A]
Hi everyone,
Does anybody know in which future version of OpenSearch Data Prepper HTTP Sink is going to be available?
Because it is documented on Github, https://github.com/opensearch-project/data-prepper/tree/main/data-prepper-plugins/http-sink, but is not available right know on 2.6.1 version (not available on official documentation of Data Prepper too)
Thanks in advance for your help.

**Comment by dlvenable** [comment]
@canob , This feature is currently not ready for use. And we have moved it to the backlog of [our roadmap](https://github.com/opensearch-project/data-prepper/projects/1).

As the feature is not yet ready, it is a good time to express any use-cases you have or features you'd like out of it. We can then be sure to incorporate those. Feel free to review https://github.com/opensearch-project/data-prepper/issues/874 and add anything you'd like.

Also, if you are interested in picking this sink up, we are happy to work you to get it completed.
